actions/campaigns.go

```diff
@@ -575,3 +576,44 @@ func GetCampaignBounces(c *gin.Context) {
 		"message": "Campaign bounces not found",
 	})
 }
+
+func DeleteCampaignSchedule(c *gin.Context) {
+	u := middleware.GetUser(c)
+
+	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"message": "Id must be an integer.",
+		})
+		return
+	}
+	campaign, err := storage.GetCampaign(c, id, u.ID)
+	if err != nil {
+		if errors.Is(err, gorm.ErrRecordNotFound) {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"message": "This campaign does not belong to you.",
```

Comment:
The error message should be the same as the one below, we don't know that this campaign actually exist or not, and the user will be misguided by this message.

Reply:
@FilipNikolovski Then we don't need this checkup